base/system/userinit/userinit.c

```diff
@@ -359,17 +372,17 @@ StartShell(VOID)
     /* Try default shell */
     if (IsConsoleShell())
     {
+        *Shell = UNICODE_NULL;
         if (GetSystemDirectoryW(Shell, ARRAYSIZE(Shell) - 8))
-            wcscat(Shell, L"\\cmd.exe");
-        else
-            wcscpy(Shell, L"cmd.exe");
+            wcscat(Shell, L"\\");
```

Comment:
Switch to strsafe while you're at it?

Reply:
In a separate commit, where all the app gets strsafe'd (or at least all this function).

Reply:
Imo it would make sense to already update all lines that you are touching anyway,
and do the rest later?